app.js

```diff
@@ -0,0 +1,221 @@
+    }
+  } else {
+    try {
+      articleLink = article["Link to Article"];
+    } catch (error) {
+      console.log(error);
+      dailyUpdatesChannel.send("An error occured, could you please try again");
+    }
+  }
+  data[category] =
+    data[category][(0, randomArticlePosition)] +
+    data[category][(randomArticlePosition + 1, lengthOfArticle)];
+  data[category] =
+    _.slice(data[category], 0, randomArticlePosition) +
+    _.slice(data[category], randomArticlePosition + 1, lengthOfArticle);
+  return articleLink;
+}
+
+// resetting schedule after changing timings
+function resetScheduler() {
+  const job = schedule.scheduleJob(rule, function () {
+    dailyUpdatesChannel.send(fetchRandomArticle("WILDCARD"));
+    cronExpression = `${rule.minute} ${rule.hour} * * ${startDay}-${endDay}`;
+  });
+}
+
+//Playing Message
+client.on("ready", async () => {
+  console.log(
+    `${client.user.username} is online on ${client.guilds.cache.size} servers!`
+  );
+
+  client.user.setActivity("Article", { type: "PLAYING" });
+
+  resetScheduler();
+});
+
+// handling on message events
+client.on("message", (msg) => {
+  dailyUpdatesChannel = msg.guild.channels.cache.find(
+    (channel) => channel.name === "readsomethinggreat"
+  );
+  cronExpression = `${rule.minute} ${rule.hour} * * ${startDay}-${endDay}`;
+
+  if (msg.author.bot) return;
+
+  if (msg.content === "get help") {
+    msg.channel.send({ embeds: [exampleEmbed] });
+  }
+
+  if (
+    msg.content.startsWith("get article") ||
+    msg.content.startsWith("Get article")
+  ) {
+    msgRecievied = msg.content.split(" ");
+    if (msgRecievied.length == 2) {
+      msg.channel.send(
+        "```Here are the categories to read upon:" +
+          "\n" +
+          "1. Wildcard" +
+          "\n" +
+          "2. Living Better" +
+          "\n" +
+          "3. Business & Tech" +
+          "\n" +
+          "4. History & Culture" +
+          "\n" +
+          "5. Science & Nature```"
+      );
+    } else {
+      if (msgRecievied.length == 3) {
+        category = _.upperCase(msgRecievied[2]);
+      } else {
+        category = msgRecievied.slice(2);
+        category = _.upperCase(category.join(" "));
+      }
+      if (categories.includes(category)) {
+        msg.channel.send(fetchRandomArticle(category)).catch((err) => {
+          console.log(err);
+          msg.channel.send("```coudn't fetch the article at the moment :( ```");
+        });
+      } else {
+        if (msgRecievied[2] == "time") {
+          msg.channel.send(
+            "The daily article will be coming " +
+              cronstrue.toString(cronExpression)
+          );
+        } else {
+          msg.channel.send(
+            "```The specified category doesn't exists. The available categories are Wildcard, Living Better, Business & Tech, History & Culture, Science & Nature```"
+          );
+        }
+      }
+    }
+  }
+
+  if (msg.content.startsWith("set article ")) {
```

Comment:
have you not updated the time input?

Reply:
yeah updated, can you see the if statements below this